tests/integration/einstein/test_operations.py

```diff
@@ -29,6 +29,7 @@
 NEW_SNAP_NAME = _cfg.tests.new_snap_name
 NOT_EXIST_IMG_NAME = _cfg.tests.not_exist_img_name
 NOT_EXIST_SNAP_NAME = _cfg.tests.not_exist_snap_name
+ING2 = _cfg.tests.IMAGE2_NAME_PARAMETER
```

Comment:
should be I**M**G2, and also you would need to put this in the config file if you are doing `_cfg.tests.something`.

But I think for the purpose of this test, you can get rid of this constant from here, and just generate a random string in the setup of your test. Something like this would work

```python
import uuid
img2 = uuid.uuid4()
```